`prepare-Changelog` shouldn't put `No new tests` if `JSTests` is updated

sosukesuzuki opened the pull request:
#### c07db62b7cad9bc1e9e51ab5d175907fa5128962
<pre>
`prepare-Changelog` shouldn&apos;t put `No new tests` if `JSTests` is updated
<a href="https://bugs.webkit.org/show_bug.cgi?id=298811">https://bugs.webkit.org/show_bug.cgi?id=298811</a>

Reviewed by Yusuke Suzuki.

This patch changes `prepare-Changelog` does not put `No new tests` if a
file is added to `JSTests`.

* Tools/Scripts/prepare-ChangeLog:
(generateFileList):

Canonical link: <a href="https://commits.webkit.org/299942@main">https://commits.webkit.org/299942@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/152b36491277a89e8be2d352fa364bf18c78bd2f

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/120825 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/40519 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/31175 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/127233 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/72901 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/122701 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/41217 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/49096 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/91773 "Passed tests") | [💥 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/61020 "An unexpected error occured. Recent messages:Printed configuration") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/123777 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/32904 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/108307 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/72471 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/31934 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/26410 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/70827 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/102400 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/26588 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/130093 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/47746 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/36258 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/100392 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/48114 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/104481 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/100295 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/25426 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/45675 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/23721 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/44422 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/47608 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/47079 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/50423 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/48763 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/09336d2f321b7744738b2fc74c621d9c1663d6c5. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50698

webkit-commit-queue commented:
Commit message contains (OOPS!), blocking PR #50698. Details: [Build #25418](https://ews-build.webkit.org/#/builders/19/builds/25418)

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/69633b352ff0a490aff40d334dc832ce512ffa0a. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50698

webkit-commit-queue commented:
Commit message contains (OOPS!), blocking PR #50698. Details: [Build #25419](https://ews-build.webkit.org/#/builders/19/builds/25419)

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/152b36491277a89e8be2d352fa364bf18c78bd2f. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50698

sosukesuzuki committed:
`prepare-Changelog` shouldn't put `No new tests` if `JSTests` is updated
https://bugs.webkit.org/show_bug.cgi?id=298811

Reviewed by Yusuke Suzuki.

This patch changes `prepare-Changelog` does not put `No new tests` if a
file is added to `JSTests`.

* Tools/Scripts/prepare-ChangeLog:
(generateFileList):

Canonical link: https://commits.webkit.org/299942@main
